redis/spring-cloud-starter-stream-sink-redis/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1 +1,2 @@
-configuration-properties.classes=org.springframework.cloud.stream.app.redis.sink.RedisSinkProperties
+configuration-properties.classes=org.springframework.cloud.stream.app.redis.sink.RedisSinkProperties, \
+  org.springframework.boot.autoconfigure.data.redis.RedisProperties
```

Comment:
I think we can have `Pool` and `Sentinel` properties here. will add while merging.